pkg/web/templates/partials/user/user_list.html

```diff
@@ -0,0 +1,47 @@
+<div>
+  <table class="table">
+    <thead class="font-bold text-base text-black">
+      <tr>
+        <th>Name</th>
+        <th>Email</th>
+        <th>Role</th>
+        <th>Created</th>
+        <th>Last Login</th>
+         <!-- TODO: Make visible to admin users only -->
+        <th>Actions</th>
+      </tr>
+    </thead>
+    <tbody>
+      {{ if .Users }}
+      {{ range .Users }}
+      <tr class="hover">
+        <td>{{ .Name }}</td>
+        <td>{{ .Email }}</td>
+        <td>{{ .Role }}</td>
+        {{ $created := .Created.Format "January 2, 2006 15:04:05" }}
+        <td>{{ $created }}</td>
+        {{ $lastLogin := .LastLogin.Format "January 2, 2006 15:04:05" }}
```

Comment:
The following error is sometimes returned when refreshing the users list: `error calling Format: value method time.Time.Format called using nil *Time pointer`

I believe there was a similar issue with a previous resource, but I couldn't quite remember how it was resolved.

Reply:
Yes, you'll have to do the if last login check on the timestamp which can be nil; not on the returned value of the Format call, e.g. 

```suggestion
        {{ if .LastLogin  }}
        {{ .LastLogin.Format "January 2, 2006 15:04:05" }}
        {{ else }}
        <td>&mdash;</td>
        {{ end }}
```

I'll make that change in the PR.